features/measurements/impl/src/main/java/org/opennms/netmgt/measurements/impl/JEXLExpressionEngine.java

```diff
@@ -99,6 +99,9 @@ public void applyExpressions(final QueryRequest request, final FetchResults resu
         final Map<String, double[]> columns = results.getColumns();
         final int numRows = timestamps.length;
 
+        // Calculate the time span
+        jexlValues.put("__diff_time", numRows < 1 ? 0d : timestamps[numRows-1] - timestamps[0]);
```

Comment:
I divide __diff_time by 1000 here instead: https://github.com/OpenNMS/backshift/blob/v1.1.2/src/Backshift.Utilities.RpnToJexlConverter.js#L145

That way all time units in the Measurement API remain in millis.